packages/shared/lib/services/connection.service.ts

```diff
@@ -523,14 +525,47 @@ class ConnectionService {
     public async listConnections(
         environment_id: number,
         connectionId?: string
-    ): Promise<{ id: number; connection_id: string; provider: string; created: string; metadata: Metadata }[]> {
+    ): Promise<{ id: number; connection_id: string; provider: string; created: string; metadata: Metadata; active_logs: ActiveLogIds }[]> {
         const queryBuilder = db.knex
             .from<Connection>(`_nango_connections`)
-            .select({ id: 'id' }, { connection_id: 'connection_id' }, { provider: 'provider_config_key' }, { created: 'created_at' }, 'metadata')
-            .where({ environment_id, deleted: false });
+            .leftJoin(ACTIVE_LOG_TABLE, function () {
+                this.on('_nango_connections.id', '=', `${ACTIVE_LOG_TABLE}.connection_id`)
+                    .andOnVal(`${ACTIVE_LOG_TABLE}.active`, true)
+                    .andOnVal(`${ACTIVE_LOG_TABLE}.type`, 'auth');
+            })
+            .select(
+                { id: '_nango_connections.id' },
+                { connection_id: '_nango_connections.connection_id' },
+                { provider: '_nango_connections.provider_config_key' },
+                { created: '_nango_connections.created_at' },
+                '_nango_connections.metadata',
+                db.knex.raw(`
+                    CASE
+                        WHEN COUNT(${ACTIVE_LOG_TABLE}.activity_log_id) = 0 THEN NULL
+                        ELSE json_build_object(
+                            'activity_log_id', ${ACTIVE_LOG_TABLE}.activity_log_id,
+                            'log_id', ${ACTIVE_LOG_TABLE}.log_id
+                        )
+                    END as active_logs
+                `)
```

Comment:
are we returning the active logs data as part of any customer  facing API endpoints?

Reply:
No, it is only internal